packages/nimble-blazor/package.json

```diff
@@ -18,10 +18,10 @@
     "format:js": "eslint . --fix",
     "test": "dotnet test -c Release",
     "pack": "cross-env-shell dotnet pack NimbleBlazor -c Release -p:PackageVersion=$npm_package_version --output dist",
-    "invoke-publish": "npm run invoke-publish:clean-nuget && npm run pack && npm run invoke-publish:nuget && npm run invoke-publish:npm -- ",
-    "invoke-publish:clean-nuget": "rimraf --glob \"dist/*.nupkg\"",
+    "invoke-publish": "invoke-publish:setup && npm run invoke-publish:nuget && npm run invoke-publish:noop -- ",
+    "invoke-publish:setup": "rimraf --glob \"dist/*.nupkg\" && npm run pack",
     "invoke-publish:nuget": "cross-env-shell dotnet nuget push \"dist/*.nupkg\" -k $NUGET_SECRET_TOKEN -s \"https://api.nuget.org/v3/index.json\"",
-    "invoke-publish:npm": "echo \"noop command to swallow npm args\"",
+    "invoke-publish:noop": "echo \"noop command to swallow npm args\"",
```

Comment:
fyi @m-akinc I'm tweaking the blazor pattern. Notice :setup merging the :clean and pack calls and renaming :npm to :noop to reflect it's behavior.

Reply:
Don't forget the `npm run` prefix for `invoke-publish:setup` like I did on the `invoke-publish` command 🙆‍♂️